src/Compilers/CSharp/Portable/Symbols/Synthesized/Records/SynthesizedRecordEqualityContractProperty.cs

```diff
@@ -16,13 +16,19 @@ internal sealed class SynthesizedRecordEqualityContractProperty : PropertySymbol
     {
         internal const string PropertyName = "EqualityContract";
 
-        public SynthesizedRecordEqualityContractProperty(NamedTypeSymbol containingType, bool isOverride)
+        public SynthesizedRecordEqualityContractProperty(NamedTypeSymbol containingType, NamedTypeSymbol? baseRecordType)
```

Comment:
>NamedTypeSymbol? baseRecordType [](start = 89, length = 31)

It feels like this parameter is unnecessary. If the base of containing type is object, it is virtual. Otherwise, it is override.

Reply:
In error cases, when overriding a non-record type, this property should not be considered an override.

---
In reply to: [446541790](https://github.com/dotnet/roslyn/pull/45336#discussion_r446541790) [](ancestors = 446541790)

Reply:
> In error cases, when overriding a non-record type, this property should not be considered an override.

I do not believe this is worth the added complexity. I am going to change this in an upcoming PR anyway.

---
In reply to: [446551544](https://github.com/dotnet/roslyn/pull/45336#discussion_r446551544) [](ancestors = 446551544,446541790)